pkg/polaris/aws/aws.go

```diff
@@ -276,10 +278,10 @@ func (a API) AccountByName(ctx context.Context, feature core.Feature, name strin
 
 // Accounts return all accounts with the specified feature matching the filter.
 // The filter can be used to search for account id, account name and role arn.
-func (a API) Accounts(ctx context.Context, feature core.Feature, filter string) ([]CloudAccount, error) {
+func (a API) Accounts(ctx context.Context, feature core.Feature, filter string, statusFilters ...core.Status) ([]CloudAccount, error) {
```

Comment:
I think there was a miscommunication. We should keep the `AccountsByStatus` function you added previously and not modify the `Accounts` function. It was just the `CloudAccountsWithFeatures` function in the `graphql` package we should extend.